src/main/resources/assets/bloodmagic/lang/de_DE.lang

```diff
@@ -9,9 +9,9 @@ item.BloodMagic.activationCrystal.creative.name=Kreativer Aktivierungskristall
 
 item.BloodMagic.sacrificialDagger.normal.name=Opfermesser
 item.BloodMagic.sacrificialDagger.creative.name=Kreatives Opfermesser
-item.BloodMagic.pack.selfSacrifice.name=Rucksack des Blutlassers
-item.BloodMagic.pack.sacrifice.name=Wappen
-item.BloodMagic.daggerOfSacrifice.name=Dolch der Opferung
+item.BloodMagic.pack.selfSacrifice.name=Weste des Blutlassens
+item.BloodMagic.pack.sacrifice.name=Vampirische Weste
+item.BloodMagic.daggerOfSacrifice.name=Schwert der Opferung
```

Comment:
If we call the sacrificial knife "Opfermesser" then we should call the sacrificial dagger "Opferdolch"